cli/coniksclient/internal/cmd/run.go

```diff
@@ -42,7 +42,10 @@ func init() {
 func run(cmd *cobra.Command, args []string) {
 	isDebugging, _ := strconv.ParseBool(cmd.Flag("debug").Value.String())
 	conf := loadConfigOrExit(cmd)
-	cc := client.New(nil, true, conf.SigningPubKey)
+
+	// FIXME: right now we're passing the initSTR, but we should really
```

Comment:
Instead of putting a FIXME here, I would like to put it in https://github.com/coniks-sys/coniks-go/blob/master/protocol/directory/directory.go#L70, where we should also store the tree etc...